clap_complete/src/dynamic/completer.rs

```diff
@@ -339,6 +352,22 @@ fn complete_arg_value(
     let mut values = Vec::new();
     debug!("complete_arg_value: arg={arg:?}, value={value:?}");
 
+    let (prefix, value) = match arg.get_value_delimiter() {
+        Some(delimiter) => match value {
+            Ok(value_os) => match value_os.rfind(delimiter) {
+                Some(pos) => {
+                    let (prefix, value) = value_os.split_at(pos);
+                    let prefix = &prefix[..prefix.len()];
```

Comment:
What is this doing?

Reply:
Sorry about this. When I first wrote the code, I got mixed up about whether the delimiter was in the `prefix` or the `value` after using `split_at`, which led to this code :)